httputil/client.go

```diff
@@ -4,10 +4,19 @@ import (
 	"io"
 	"net/http"
 	"strings"
+	"time"
 )
 
 const (
 	headerXForwardedFor = "X-Forwarded-For"
+
+	// default transport settings
```

Comment:
This comment isn't following the usual convention. It should start with the constant name. These constants are already pretty well documented so we either remove all comments or add for all of them like on line 18:

```go
	// DefaultRequestTimeout is the default timeout for HTTP requests for default HttpClient.
	DefaultRequestTimeout = 30 * time.Second
```